packages/ragbits-document-search/src/ragbits/document_search/ingestion/providers/base.py

```diff
@@ -13,7 +13,7 @@ class DocumentTypeNotSupportedError(Exception):
     """
 
     def __init__(self, provider_name: str, document_type: DocumentType) -> None:
-        message = f"Document type {document_type} is not supported by the {provider_name}"
+        message = f"Document type {document_type.value} is not supported by the {provider_name}"
```

Comment:
This tripped tests on some newer Python versions due to change in how enum values are displayed in newer Python version. That part makes sense. What I don't understand is why the problem only started to appear in CI in this PR.